apps/checker/app/matchers/DictionaryMatcher.scala

```diff
@@ -59,40 +59,45 @@ class DictionaryMatcher(
       entities: List[EntityInText]
   ): Boolean = {
     entities.exists(entity =>
-      entity.range.from == ruleMatch.fromPos && entity.range.to == ruleMatch.toPos
+      entity.range.from <= ruleMatch.fromPos && entity.range.to >= ruleMatch.toPos
```

Comment:
Is this method still needed? (Perhaps, given comment below 😀  – but I think we should only need one?)

Reply:
We're using it again now!